### PR TITLE
[RIO-2024] New Event 2024

### PR DESCRIPTION
**Pull request by mauriciopgomes**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

**Comment by mauriciopgomes** [comment]
Add No-merge label please

**Comment by adrianmoisey** [comment]
> Add No-merge label please

I've done.
Let us know when you're ready for a review and we can remove it

**Comment by mauriciopgomes** [comment]
everything is ok, can you do merge? please

File: content/events/2022-rio-de-janeiro/conduct-fr.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 +++
 Title = "Conduct"
 Type = "event"
-Description = "Code de conduite devopsdays Rio de Janeiro 2022"
+Description = "Code de conduite DevopsDays Rio de Janeiro 2022"
```
Comment (glasnt) [inline content/events/2022-rio-de-janeiro/conduct-fr.md:4]: You have a number of these capitalization changes. Could you please remove these from this PR, and keep this PR to just the new event? Thank you!

**Comment by mauriciopgomes** [comment]
Done

File: data/events/2024-rio-de-janeiro.yml [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+# registration_date_start: 2024-04-12T00:00:00-03:00 # start accepting registration. Leave blank if registration is not open yet. This will make the "Register" button appear on your "Welcome" page.
+# registration_date_end: 2024-08-17T00:00:00-03:00 # close registration. Leave blank if registration is not open yet. If you set "registration_date_start" you need a value here.
+
+registration_closed: "false" #set this to true if you need to manually close registration before your registration end date
+# registration_link: "https://www.sympla.com.br/evento/devopsdays-rio-de-janeiro-2024/1929984" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+# coordinates: "-22.997961, -43.345923" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+# location: "MHS Eventos"
+# location_address: "Av. das Américas, 3434 - Barra da Tijuca, Rio de Janeiro" #Optional - use the street address of your venue. This will show up on the welcome page if set.
```
Comment (glasnt) [inline data/events/2024-rio-de-janeiro.yml:33]: It looks like you've copied the content from your 2023 event, which includes venue but also a program and potentially speakers. 

Can you please use the [new event script](https://github.com/devopsdays/devopsdays-web/tree/main/utilities#events) to make sure you're not accidentally carrying over information, and you're using the current best practice template for your event? Thanks!

Comment (mauriciopgomes) [reply data/events/2024-rio-de-janeiro.yml]: Done

**Pull request merged**